hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -616,6 +616,9 @@ private HoodieTableMetaClient initializeMetaClient() throws IOException {
    * @return List consisting of {@code DirectoryInfo} for each partition found.
    */
   private List<DirectoryInfo> listAllPartitionsFromFilesystem(String initializationTime) {
+    if (dataWriteConfig.getMetadataConfig().isFileSystemBootstrapDisabled()) {
```

Comment:
food for thought : I think this should prevent any MDT partitions from being initialized, not just `files`

Reply:
Do the partitions need to be initialized as empty? Simply skipping over the method that calls this lead to errors.